lua/starfall/sflib.lua

```diff
@@ -102,6 +102,8 @@ function SF.throw ( msg, level, uncatchable )
 	error( err )
 end
 
+local real_type = function () end
```

Comment:
Just quickly tested this using collectgarbage("count") before and after assignment. Memory for a function is ~0.02 and for an empty object it's ~0.03. For primitives print just outputs a 0, but equivalence with strings and numbers is obviously not unique, and risky. So out of the unique variables we can choose, empty function seems like the best bet.